### PR TITLE
Add mitigating emergency doc

### PR DESCRIPTION
**Pull request by lisaej**

### CODE REVIEW

**Comment by lisaej** [comment]
Not sure who to ask for review/merge?

File: team-culture/emergency.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Mitigating emergencies
+
+1. How do we better define what constitutes an emergency and what doesn’t, and communicate this definition to our stakeholders?
+
+Emergencies are:
+Issues that block or disrupt the overall viewing / functions of Fusion.net
+ex.. fusion.net is down, site hacked/ security breach, backend is frozen for ALL writers…
```
Comment (danielbachhuber) [inline team-culture/emergency.md]: Could we be more specific about these example emergencies?

You could potentially create a table in this page which has examples of emergencies in one column and non-emergencies in another column.

File: best-practices/emergencies.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Emergencies are:
+Issues that block or disrupt the overall viewing / functions of Fusion.net
+ex.. fusion.net is down, site hacked/ security breach, backend is frozen for ALL writers…
+
+Emergencies are NOT:
+Things that seem out of place or slightly off for a single post or page. These are bugs please make note of them in the Edit-Tech room. This includes display issues on video/article pages, Minor CSS tweaks, select video not playing, embed issues, being locked out of your WP account.
+
+New feature requests or undefined product requests are not emergencies and should go through the process of creating a Trello card to properly document requirements and assign people.
+
```
Comment (danielbachhuber) [inline best-practices/emergencies.md]: Worth mentioning that a good way to initiate the process is to ask in `#edit-tech`

File: best-practices/emergencies.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+2. How can we better include stakeholders in the QA process so we don’t miss blockers?
+
+Note when projects are beginning the stakeholders and make sure they are aware with the changes being made.
+
+3. How should we make sure we have appropriate tech support coverage outside of normal business hours?
+
+Communicate with Editorial about any specific events we might be holding outside business hours and have 1 engineer and 1 prod manager aware of the event and in a position where if need be they can assist with any problems.
```
Comment (danielbachhuber) [inline best-practices/emergencies.md]: Probably want to change "Editorial" to "stakeholders".

**Pull request merged**